data/json/items/melee/swords_and_blades.json

```diff
@@ -1509,7 +1509,7 @@
     "symbol": "/",
     "color": "light_gray",
     "name": { "str": "rapier" },
-    "description": "This is a blunted, cheap replica of a thin sword with an ornate hand guard.  It looks like the preferred weapon of gentlemen and swashbucklers.  Light and quick, it makes any battle a stylish battle.",
+    "description": "This is a blunted, cheap replica of a thin sword with an ornate handguard.  It looks like the preferred weapon of gentlemen and swashbucklers.  Light and quick, it makes any battle a stylish battle.",
```

Comment:
```suggestion
    "description": "This is a blunted, cheap replica of a thin sword with an ornate handguard.  It looks like the preferred weapon of gentlemen and swashbucklers.  Light and quick, it makes any battle stylish.",
```

Reply:
I think the repetition here serves well. If you imagine the word 'stylish' being italicized if this were presented in prose, I think it works better.